wolox-books/src/app/app.module.ts

```diff
@@ -3,21 +3,23 @@ import { NgModule } from '@angular/core';
 
 import { Router } from '@angular/router';
 import { AppRoutingModule } from './app-routing.module';
+import { AuthGuard } from './auth.guard';
+import { UnauthGuard } from './unauth.guard';
 
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { HttpClientModule } from '@angular/common/http';
 
 import { AppComponent } from './app.component';
-import { RegisterComponent } from './screens/register/register.component';
-import { LoginComponent } from './screens/login/login.component';
-import { AuthComponent } from './screens/auth/auth.component';
+import { RegisterComponent } from './screens/unauth/register/register.component';
```

Comment:
you can have all the components at the same level without having this auth and unauth folder